testsuite/rust-bridged/README.md

```diff
@@ -0,0 +1,5 @@
+This test checks the shared target functionality of Rust, by having a submission and checker in Rust.
+
+The first case always runs fine, since the checker will not be compiled until after the submission is run.
+
+The second case will IR if the submission and checker have misconfigured target directories.
```

Comment:
In vim speak, what I want is `gggqG`.

(Break lines at 79 columns even if it breaks a sentence/paragraph into two. That's the standard formatting for this kind of thing.)

Reply:
Done.